<!-- <launch>
  <node name="EventMessageEditor" pkg="events_repacking_helper" type="EventMessageEditor" output="screen" required="true"
    args="/home/zhouyi/workspace/dataset/DVS/hkust_HDR/2020-07-14-12-38-42.bag
          /home/zhouyi/workspace/dataset/DVS/hkust_HDR/hahaha.bag">
  </node>
</launch> -->

<launch>
  <param name="bag_in" value="$(arg bag_in)"/>
  <param name="bag_out" value="$(arg bag_out)"/>
  <node pkg="events_repacking_helper" type="EventMessageEditor" name="EventMessageEditor" output="screen" required="true"/>
</launch>
